<?xml version="1.0"?>

<launch>
  <node name="filter_lidar" pkg="igvc" type="filter_lidar" output="screen">
      <param name="x_offset" value=".29"/>
      <param name="y_offset" value=".03"/>
      <param name="x_size" value=".4"/>
      <param name="y_size" value=".4"/>
      <param name="max_dist" value="15"/>
      <param name="back_buffer" value =".61"/>
    </node>
</launch>
